client/flutter/lib/pages/settings_page.dart

```diff
@@ -74,9 +68,48 @@ class _SettingsPageState extends State<SettingsPage> {
                   style: Theme.of(context).textTheme.subhead,
                 ),
               ),
-
-              /// TODO: Implement UI:-
-              /// TODO:   selection of language preferences already created PR for it (#654)
+              Padding(
+                padding: const EdgeInsets.all(15.0),
+                child: SelectableText(
```

Comment:
Why `SelectableText` instead of `Text`?

Reply:
So, it can be selected (just for good practice) when we deploy to web in the future.

Reply:
@britannio Sorry for the merge conflicts texts.

Reply:
I don't know of plans to target web in the future, maybe @advayDev1 can confirm?

Reply:
![Screenshot_20200331-212858__01](https://user-images.githubusercontent.com/33752528/78072546-384c4580-7397-11ea-9a8b-fc29601aace4.jpg)
Also, the constrast is wrong on my device.

Reply:
The reason behind that bug isn't your device, it's `accentColor` in app theme. I think accentColor should not be closer to `white` if our default theme is `white`. Flutter has separate themes for fonts `textTheme` so, I think we've to use that. Until then, I could just replace `SelectableText` with `Text` widgets for now. @britannio

Reply:
Using a `Text` widget solves the issue

Reply:
Well, yeah but I strongly suggest to use built-in TextTheme for styling texts in the app.